db/seeds.rb

```diff
@@ -1,7 +1,74 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+ryto = User.create(
+  email: 'ryto@gmail.com',
+  username: 'ryto',
+  password: 'password',
+  first_name: 'Ryto',
+  last_name: 'Verkar'
+)
+
+mike = User.create(
+  email: 'mike@gmail.com',
+  username: 'mike',
+  password: 'password',
+  first_name: 'Mike',
+  last_name: 'Jordan'
+)
+
+anna = User.create(
+  email: 'anna@gmail.com',
+  username: 'anna',
+  password: 'password',
+  first_name: 'Anna',
+  last_name: 'Banana'
+)
+
+george = User.create(
+  email: 'george@gmail.com',
+  username: 'george',
+  password: 'password',
+  first_name: 'George',
+  last_name: 'Washington'
+)
+
+ryto.active_friendships.create(passive_friend: anna, confirmed: true)
+mike.active_friendships.create(passive_friend: ryto, confirmed: true)
+anna.active_friendships.create(passive_friend: mike)
+george.active_friendships.create(passive_friend: anna)
+
+50.times do |n|
```

Comment:
Do you really need as many objects in seeds?

Reply:
Hahaha.. I wanted to make sure I have lots of signups right off the bat :D :D :D